.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Documentation
+about: Update the documentation
+title: 'Documentation: '
+labels: 'alert: NEED ACCOUNT KEY, alert: NEED MORE DEFINITION, alert: NEED CYCLE ASSIGNMENT, type: documentation'
+assignees: ''
+
+---
+
+*Replace italics below with details for this issue.*
+
+## Describe the Task ##
+*Provide a description of the task here.*
+
+### Time Estimate ###
+*Estimate the amount of work required here.*
+*Issues should represent approximately 1 to 3 days of work.*
+
+### Sub-Issues ###
+Consider breaking the task down into sub-issues.
+- [ ] *Add a checkbox for each sub-issue here.*
+
+### Relevant Deadlines ###
+*List relevant project deadlines here or state NONE.*
+
+### Funding Source ###
+*Define the source of funding and account keys here or state NONE.*
+
+## Define the Metadata ##
+
+### Assignee ###
+- [ ] Select **engineer(s)** or **no engineer** required
```

Comment:
```suggestion
- [ ] Select appropriate **assignee** for this issue
```
Having an engineer AND scientist assigned for doc updates is overkill. Instead, just remind them to think about who should work on it.

Reply:
I made this change.